src/domains/dashboard/components/Wallets/Wallets.tsx

```diff
@@ -46,7 +48,7 @@ export const Wallets = ({
 			.wallets()
 			.values()
 			.filter((wallet) => wallet.network().isLive());
-	}, [activeProfile]);
+	}, [activeProfile, walletsCount]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Why is eslint disabled for this line?

Reply:
Because needed to re-run this hook each time wallets count changes, though I am not using this variable inside the hook and it complains about unnecessary dep.